ui/pages/send/send-footer/send-footer.stories.js

```diff
@@ -1,30 +1,37 @@
 import React from 'react';
-import { action } from '@storybook/addon-actions';
-import { boolean } from '@storybook/addon-knobs';
 
 import SendFooter from './send-footer.component';
 
 export default {
   title: 'Pages/Send/SendFooter',
   id: __filename,
+  argTypes: {
+    clearSend: { action: 'Cancel Button Pressed' },
+    sign: { action: 'Next Button Pressed' },
+    from: { control: 'object' },
+    disabled: { control: 'boolean' },
+    mostRecentOverviewPage: { control: 'text' },
+    sendErrors: { control: 'object' },
```

Comment:
I think we can probably add the other function type props as actions(even though they say they're just to make the component show no error)
```suggestion
    sendErrors: { control: 'object' },
    history: { action: 'history' },
    addToAddressBookIfNew: { action: 'addToAddressBookIfNew' },
    resetSendState: { action: 'resetSendState' },
```

Reply:
done